tests/test_api.py

```diff
@@ -1345,7 +1345,7 @@ def test_create_build_config_create(self):
             .should_receive('create_build_config')
             .with_args(json.dumps(build_json))
             .once()
-            .and_return(flexmock(json=lambda: {'spam': 'maps'})))
+            .and_return(flexmock(json=lambda: {'spam': 'maps', 'spec': {}})))
```

Comment:
Not sure why this change is required - nothing in current PR changes the output of `create_build_config`